tests/time_tests/test_runner/conftest.py

```diff
@@ -114,6 +115,9 @@ def cl_cache_dir(pytestconfig):
     More: https://github.com/intel/compute-runtime/blob/master/opencl/doc/FAQ.md#how-can-cl_cache-be-enabled
     """
     cl_cache_dir = pytestconfig.invocation_dir / "cl_cache"
+    # if cl_cache generation to a local `cl_cache` folder doesn't work, specify
+    # `cl_cache_dir` environment variable in an attempt to fix it (Linux specific)
+    os.environ["cl_cache_dir"] = str(cl_cache_dir)
```

Comment:
Check it works on Win (general purpose doesn't work with https://github.com/intel/compute-runtime/releases/tag/19.41.14441)

Reply:
Checked - it works